packages/vm/src/runBlock.ts

```diff
@@ -558,16 +559,30 @@ export function calculateMinerReward(minerReward: bigint, ommersNum: number): bi
 }
 
 export async function rewardAccount(
+  common: Common,
```

Comment:
Short nit: can we make this „a bit less breaking“ (function might’ve in direct usage) and put common last and make it optional (and the do „if common and common.isActivated…“ checks)?

Then we can mention this in release not docs (that people should have a look if they use this method directly and optimally switch to pass a common there) and code is not directly breaking.

Reply:
Just seeing this now. I agree. I will put `common` last and make it optional.